bioio_imageio/tests/test_reader.py

```diff
@@ -15,10 +15,10 @@
 @pytest.mark.parametrize(
     "filename, set_scene, expected_shape, expected_dims_order",
     [
-        ("example.bmp", "Image:0", (480, 640, 4), "YXS"),
+        ("example.bmp", "Image:0", (480, 640, 3), "YXS"),
         ("example.png", "Image:0", (800, 537, 4), "YXS"),
         ("example.jpg", "Image:0", (452, 400, 3), "YXS"),
-        ("example.gif", "Image:0", (72, 268, 268, 4), "TYXS"),
+        ("example.gif", "Image:0", (72, 268, 268, 3), "TYXS"),
```

Comment:
for example.gif we have an issue with the reader here. 
```
            elif image_length > 1:
                # Read and stack all frames
                frames = []
                for frame in reader:
                    frames.append(frame)

                image_data = np.stack(frames)
``` 
We have a first frame that is uneven
![locals](https://github.com/user-attachments/assets/2602fe69-1fc0-46ad-ba53-72c65f61bb57)

Reply:
@evamaxfield @toloudis any thoughts here? np.stack expects same-size arrays, not sure why the first one is missing a dim.

Reply:
I found this as a result of me upgrading the various base imageio libs. I don't think we changed anything, I think the underlying imageio library changed. I changed the test as a part of just fiddling around with stuff to figure out what was happening. I think we may just want to rollback the imageio upgrade.